src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/MirrorFetchRequest.java

```diff
@@ -0,0 +1,42 @@
+package com.atlassian.bitbucket.jenkins.internal.scm;
+
+class MirrorFetchRequest {
+
+    private final String serverId;
+    private final String jobCredentials;
```

Comment:
Everywhere else in the code we have called this `credentialsId`. Any reason this has been changed here? I think we should stick to one term.

Reply:
Fixed. I have used Job Credentials since simple `credentialsId` could be either job or global credentials. I have renamed it to retain consistency.